### ISSUE TITLE
Limiter Auto-Leveling

### ISSUE THREAD

**Issue by Digitalone1** [wwmm/pulseeffects#732]
I discovered that calf limiter has a bool option called auto-level:

```
auto-level          : Auto-level
                        flags: leggibile, scrivibile, controllabile
                        Boolean. Default: false
```

Looking the limiter code, it's specified that the plugin is doing *automatic makeup gain*, so we reduce the output level accordingly binding both `output-gain` and `limit` to the same gsettings key.

But from version 0.90.2 this new option was introduced that disables auto-level by default. See changelog [here](https://freshcode.club/projects/calf).

```
Limiter: Allow toggling auto-leveling. . . . Move all limiter auto-level toggles to header.
```

So this behavior is not needed anymore and we should expose this toggle in the UI (along with the output gain) to let the user enable auto leveling, right?

Besided, I was looking the gsettings schema and sourceoutput limiter has other unused options I don't know:

```
        <key name="autovolume-state" type="b">
            <default>false</default>
        </key>
        <key name="autovolume-window" type="d">
            <range min="1" max="1000"/>
            <default>1000.00</default>
        </key>
        <key name="autovolume-target" type="i">
            <range min="-99" max="-4"/>
            <default>-12</default>
        </key>
        <key name="autovolume-tolerance" type="i">
            <range min="1" max="4"/>
            <default>1</default>
        </key>
        <key name="autovolume-threshold" type="i">
            <range min="-99" max="0"/>
            <default>-50</default>
        </key>
```

Was the limiter UI using a different plugin in the past?

**Comment by wwmm** [comment]
>So this behavior is not needed anymore and we should expose this toggle in the UI (along with the output gain) to let the user enable auto leveling, right?

I did not notice this control addition. Now it is better to show it in the interface and letting the user decide what he/she wants.

>Was the limiter UI using a different plugin in the past?

My first attempts to implement an autogain were very simple. Based on the limiter output I adjusted its input gain. So the autogain controls were in the the limiter interface. Eventually I learned to make GStreamer plugins and the autogain was moved to its own place. But it seems I forgot to remove the old keys from this file.
